article/_posts/2022-02-22-scanner-invariant-representations.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: review
+title: "Scanner invariant representations for diffusion MRI harmonization"
+tags: dMRI domain-adaptation
+author: "Carl Lemaire"
+cite:
+    authors: "Daniel Moyer, Greg Ver Steeg, Chantal M. W. Tax, Paul M. Thompson"
+    title:   "Scanner invariant representations for diffusion MRI harmonization"
+    venue:   "Magn Reson Med. 2020; 84: 2174–2189"
+pdf: "https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28243"
+---
+
+# Introduction
+
+dMRI acquisitions from different hospitals can be different for a same patient, because of e.g. the brand of the scanner and the scanning protocol. We would like to harmonize acquisitions from different sites. In this approach, a VAE is trained, such that the decoding is conditionned on a site; this allows to convert images from one site to another. This method allows to harmonize data by choosing a reference site and converting all images to it.
```

Comment:
"(...) can be different": sounds vague. What about "(...) can provide different image values for the same locations"?